test/integration/sapi-policy-authorizeNV.int.c

```diff
@@ -228,6 +229,30 @@ test_invoke (TSS2_SYS_CONTEXT *sapi_context)
                                                  &data_out,
                                                  &iv_out,
                                                  &rsp_auth));
+    if (rc == TPM2_RC_COMMAND_CODE) {
+        LOG_WARNING("Encrypt/Decrypt not supported by TPM");
+        rc = Tss2_Sys_NV_UndefineSpace(sapi_context,
+                                       TPM2_RH_OWNER,
+                                       nv_index,
+                                       &cmd_auth,
+                                       &rsp_auth);
+        if (rc != TSS2_RC_SUCCESS) {
+            LOG_ERROR("Tss2_Sys_NV_UndefineSpace failed: 0x%"PRIx32, rc);
+            return 99; /* fatal error */
```

Comment:
I wouldn't think this is an hard error? Perhaps return EXIT_FAILURE would be better here and below. It will also be be more consistent with the error codes returned elsewhere in this file. Later we should also change all the exit(1); to return EXIT_FAILURE;

Reply:
This is what for example the sapi-encrypt-decrypt tests do in the same place, I just tried to be consistent with that. Should all of those be changed as well?

Reply:
Let's keep these consistent within the same file at least.  The sapi-encrypt-decrypt test can be changed later.